src/plugins/controls/public/options_list/components/options_list_popover_footer.tsx

```diff
@@ -54,7 +54,11 @@ export const OptionsListPopoverFooter = ({ isLoading }: { isLoading: boolean })
       >
         {isLoading && (
           <div style={{ position: 'absolute', width: '100%' }}>
-            <EuiProgress size="xs" color="accent" />
+            <EuiProgress
+              data-test-subj="optionsList-control-popover-loading"
```

Comment:
The `optionsListPopoverWaitForLoading` method for the controls page object was checking for this `data-test-subj`, but it didn't actually exist. SO, now it does 💃